SPECS/libatomic_ops/libatomic_ops.spec

```diff
@@ -1,77 +1,90 @@
-Summary:    Atomic memory update operations portable implementation
-Name:       libatomic_ops
-Version:    7.6.6
-Release:        4%{?dist}
-License:    GPLv2 and MIT
-URL:        https://github.com/ivmai/libatomic_ops
-Group:      Development/Libraries
-Source0:    http://www.ivmaisoft.com/_bin/atomic_ops/libatomic_ops-%{version}.tar.gz
+Summary:        Atomic memory update operations portable implementation
+Name:           libatomic_ops
+Version:        7.6.12
+Release:        1%{?dist}
+License:        GPLv2 and MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Development/Libraries
+URL:            https://github.com/ivmai/libatomic_ops
+Source0:        https://github.com/ivmai/libatomic_ops/releases/download/v%{version}/%{name}-%{version}.tar.gz
 
 %description
 This package provides semi-portable access to hardware-provided atomic memory update operations on a number of architectures.
 
-%package devel
-Summary:    Development files for the libatomic_ops library
-Group:      Development/Libraries
-Requires:   libatomic_ops
-Provides:   libatomic_ops-devel
-Provides:   libatomic_ops-devel(x86-64)
+%package        devel
+Summary:        Development files for the libatomic_ops library
+Group:          Development/Libraries
+Requires:       %{name} = %{version}-%{release}
 
 %description devel
-Libraries and header files for libatomic_ops library.
+Libraries and header files for the libatomic_ops library.
 
+%package        static
+Summary:        Static libraries for libatomic_ops
+Requires:       %{name}-devel = %{version}-%{release}
+
+%description static
+Static libraries for the libatomic_ops library
 
 %prep
-%setup -q
-%build
-./configure --prefix=%{_prefix}      \
```

Comment:
Wondering if we missed `%{_prefix}` for configure ?

Reply:
I believe this is included inside the `%configure` macro.

Reply:
Pawel is correct here- the macro should handle setting the default prefix.